boards/beagle/beaglev_fire/doc/index.rst

```diff
@@ -22,11 +22,18 @@ hobbyists, and researchers to explore and experiment with RISC-V technology.
 Building
 ========
 
+BeagleV-Fire supports 3 board targets in Zephyr:
+1. beaglev_fire/polarfire/e51
+2. beaglev_fire/polarfire/u54
+3. beaglev_fire/polarfire/u54/smp
+
+More information regarding the differences between the cores can be found in the [datasheet](https://ww1.microchip.com/downloads/aemDocuments/documents/FPGA/ProductDocuments/ProductBrief/PolarFire-SoC-Product-Overview-60001656.pdf)
+
 Applications for the ``beaglev_fire`` board configuration can be built as usual:
 
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
-   :board: beaglev_fire
+   :board: beaglev_fire/polarfire/{e51,u54,u54/smp}
    :goals: build
```

Comment:
I would recommend to keep the example command one that can be copy-pasted and "just works". So just show one example, e.g. for E51, and change to "Applications can be built as usual, for example when targeting the E51 core:"